CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Maintenance mode for the ASB.
+  When started with `--resume-only` the ASB does not accept new, incoming swap requests but only finishes swaps that are resumed upon startup.
+
+### Fixed
+
+- The ASB returns proper error messages for expected errors upon starting a swap.
+  This is a breaking change because the spot-price protocol response changed.
+  Expected errors scenarios that are now reported back to the CLI:
+  1. Balance of ASB too low
+  2. Buy amount sent by CLI exceeds maximum buy amount accepted by ASB
+  3. ASB is running in maintenance mode and does not accept incoming swap requests
```

Comment:
What do you think about collapsing these? The added "maintenance mode" motivated adding errors to the network layer so I think we could mentioned them as one change.

Reply:
I gave it a try and find it better separate, mainly because one only concerns the ASB and the other concerns mostly the CLI. 
The fact that we introduced the error messages as part of the `resume-only` mode is a technicality that is irrelevant for users.